docs/_events/super_navigation_header.md

```diff
@@ -22,26 +27,35 @@ events:
         value: header menu bar
       - name: text
         value: Menu/Search
-  - name: GOV.UK logo
-    description:
+  - name: link clicks
+    description: Triggered when the GOVUK logo, a header "menu" section link, or a link under the search component is clicked, right clicked, shift clicked, control clicked, or windows key/command key clicked. Note that this is currently implemented with three index values e.g. "1.1.1". This is noted in our technical debt documentation.
```

Comment:
I have a PR in review to change this, so might be worth removing the bit about "1.1.1" here.

Reply:
@andysellick Thanks, removed.